### DISCUSSION TITLE
`getListingProperties` always returns empty arrays

### DISCUSSION THREAD

**Discussion by jakeleventhal** [etsy/open-api#1110] [Q&A] [answered]
My listings have multiple variants, yet for some reason, I always get `getListingProperties` to show an empty array. Why is this?

**Comment by emergingdzns** [comment, marked as answer]
There is a difference between properties and variants. In Etsy, properties are descriptive items that do not affect variations or anything like that. For instance, you might have a single color as a property. But your variants might be based on size and shape. If you want the properties that define your variations, you have to call the listing inventory endpoint. Otherwise you're just getting non-variant properties.

The number of variants that you can have is limited. That's why properties are used for those things that don't affect variations.
